src/cards/Article.jsx

```diff
@@ -1,2 +1,20 @@
-function Article() {}
+import Button from "../styleComponents/Button";
+
+function Article({ children }) {
```

Comment:
You will have a single article to deal with for the next task, naming could get confusing. Maybe change this to ArticleCard?